src/nekontrol/interactive/cli.py

```diff
@@ -16,26 +15,24 @@ def cli():
     ...
 
 
-@cli.command(context_settings={"help_option_names": ["-h", "--help"]})
+@cli.command("test", context_settings={"help_option_names": ["-h", "--help"]})
 @click.argument("file-path", metavar="FILE", type=executable_file)
 @click.option("-p", "--problem", type=str, help="The kattis problem name")
 @click.option("-d", "--diff", type=bool)
 @click.option("-c", "--color", type=bool)
 @click.option("-v", "--verbose", type=bool)
 @click.option("--ignore-debug", type=bool)
-def test(
+def test_cli(
```

Comment:
I think it would be preferable to have a directory structure like `interactive/commands` or something along those lines, so we know which files contain commands and which don't. Then we don't have to name them in this way either.